src/test/java/com/suse/saltstack/netapi/client/SaltStackClientTest.java

```diff
@@ -706,6 +714,38 @@ public void testJobsAsync() throws Exception {
     }
 
     @Test
+    public void testJobsPending() throws Exception {
+        stubFor(any(urlMatching(".*"))
+                .willReturn(aResponse()
+                .withStatus(HttpURLConnection.HTTP_OK)
+                .withHeader("Content-Type", "application/json")
+                .withBody(JSON_JOBS_RESPONSE_PENDING)));
+
+        ResultInfoSet result_set = client.getJobResult("some-job-id");
```

Comment:
This should be all done now. I'm too used to using lower_case for variables and camelCase for functions and CamelCase for other types :smile: